packages/components/card/src/EntryCard/EntryCard.tsx

```diff
@@ -56,7 +56,7 @@ function EntryCardDescription({
   const truncatedDescription = truncate(description, 95, {});
```

Comment:
Is this not working or what?

Reply:
it is, but the problem is actually `isTruncated` prop. as far as I understand they are doing the same thing, and we need to break-word to handle this case.